test/rubygems/test_gem_commands_push_command.rb

```diff
@@ -93,6 +93,28 @@ def test_execute_host
     assert_equal Gem.read_binary(@path), @fetcher.last_request.body
     assert_equal "application/octet-stream",
                  @fetcher.last_request["Content-Type"]
+
+  end
+
+  def test_execute_allowed_push_host
+    @spec, @path = util_gem "freebird", "1.0.1" do |spec|
+      spec.metadata['allowed_push_host'] = "https://privategemserver.example"
```

Comment:
Just FYI, in previous commit `"` was used even for string literals which don't
need interpolation. I would favor `'` in such case, but I don't know what is
the preferred style in RubyGems (current code on `master` uses both). I'm just
reporting the inconsistancy in those new changes.

Reply:
why would you favor `'` in such case?

Reply:
I personally find it easier/faster when reading (no need to read between two
`'` to check for code) and only little additional work when writing.
Using `"` everywhere is interesting too because it eliminates the question and
this difference. I use this last coding style too almost every day without any
problem, but I have a slight preference for first one.

However when seeing two "different" coding styles on a single line of code, I
thought it was unintentional, that's why I commented.